src/roadmap/data/systems.py

```diff
@@ -191,7 +195,7 @@
         major=9,
         minor=1,
         start_date=date(2022, 11, 15),
-        end_date=date(2023, 5, 10),
+        end_date=date(2023, 5, 31),
```

Comment:
same here, this is expected?

Reply:
Same thing here but for [RHEL 9](https://access.redhat.com/articles/4038291#red-hat-enterprise-linux-9-8).

Reply:
When I put in the end dates, if I wasn't sure I just made it line up with the next dot release.